### DISCUSSION TITLE
transmission for Solaris

### DISCUSSION THREAD

**Discussion by victorbelous** [transmission/transmission#4599] [General]
Hi!
I use for a long time transmission

bvictor@Enot14:17:07~$transmission-daemon -v
Transmission 1.93 (10621)  http://www.transmissionbt.com/
A fast and easy BitTorrent client

on

bvictor@Enot14:27:17~$uname -a
SunOS Enot 5.11 11.3 i86pc i386 i86pc

Is it possible to compile for the Solaris 11.3 a last version of transmission?
Thanks
Victor

**Comment by mikedld** [comment]
There're 3 related build configurations on TeamCity:
* [OpenIndiana (GCC)](https://trtc.mikedld.com/buildConfiguration/Transmission_Sanity_OpenIndianaX86) — works
* [Solaris (GCC)](https://trtc.mikedld.com/buildConfiguration/Transmission_Sanity_SolarisAmd64) — works
* [Solaris (SunPro)](https://trtc.mikedld.com/buildConfiguration/Transmission_Sanity_SolarisX86SunPro) — disabled since November 2019 (outdated compiler)

Is there a particular issue that you're facing? What have you tried so far and what doesn't work?

**Reply by victorbelous** [reply]
Thanks, Mikedld
But I don't have a login on trtc.mikedld.com site. I entered as a quest and can't to download the Solaris build. How I can download it?

**Reply by mikedld** [reply]
Those aren't release builds, and I didn't mean to suggest you to download/use them. I was merely pointing out (answering your question) that compiling for Solaris is possible and expecting you to do that on your end, yourself.

**Comment by victorbelous** [comment]
I am sorry, I find the download picture
